frontend/benefit/applicant/src/components/header/useHeader.ts

```diff
@@ -50,7 +50,7 @@ const useHeader = (): ExtendedComponentProps => {
   );
 
   const { data: application } = useApplicationQuery(id);
-  const canWriteNewMessages = !application?.batch;
+  const canWriteNewMessages = !application?.has_batch;
```

Comment:
This would require type changes. I'd suggest you use `batch` like before, not `has_batch`. Or is that namespace causing an issue?

Reply:
Oh I forgot to add that `has_batch` to the Application's frontend type.  For some reason VScode did not give an error. The main problem is indeed the batch attribute namespace is referring to a boolean value in the applicants context and to a Dict value in the handlers context and the validation did not previously complain  about it as it got a None (null) value.